CDS/scripts/dailymotion_upload.pl

```diff
@@ -195,37 +195,73 @@
 
 if ($block == 0) {
```

Comment:
what does $block actually represent?

Reply:
Weather or not to ban mobile access to the video. 

It set to 0 then do not block. If set to 1 then block. Currently does not do anything because someone told me to comment out the line which actually makes the code block mobile access.